apps/batch-script/batch-script-buildpack/bin/build.bat

```diff
@@ -8,6 +8,7 @@ set layers_dir=%1
 echo [[processes]]       >> %layers_dir%\launch.toml
 echo type = "web"        >> %layers_dir%\launch.toml
 echo command = ["app.bat"] >> %layers_dir%\launch.toml
+default = true
```

Comment:
@Srishti-j18 you need to write the value `default = true` into the `launch.toml` file, just follow the same way line 10 with the `echo` command

Reply:
You also need to do the equivalent for the [bash](https://github.com/buildpacks/samples/blob/main/apps/bash-script/bash-script-buildpack/bin/build#L13) buildpack